README.md

```diff
@@ -108,6 +108,12 @@ solr.baseUrl=http://localhost:8983/solr/
 | jms.password | JMS password | no | null |
 | jms.connections | The JMS connection count | no | 10 |
 | jms.consumers | The JMS consumer count | no | 1 |
+| Fixity Service |
+| fixity.enabled | Enables/disabless fixity service  | no | false |
+| fixity.input.stream | Fixity Service message input stream | no | broker:topic:fixity |
+| fixity.delay | A delay in milliseconds between each fixity check to reduce load on server | no | 0 |
+| fixity.success|  It is also possible to trigger an action on success (by default, this is a no-op) | no | null |
```

Comment:
Would be helpful to specify that this is expecting a camel route action so people know what syntax to use.

Reply:
will do.